upgrades/3.x/README.adoc

```diff
@@ -1,6 +1,9 @@
 = Upgrade Notes
 
 ifdef::env-github[]
+
+include::3.16.0/README.adoc[leveloffset=+1]
```

Comment:
should also be added line 96 (for the import in documentation website)

Reply:
:heavy_check_mark: